package.json

```diff
@@ -60,6 +61,7 @@
     "@babel/core": "^7.10.3",
     "@babel/preset-env": "^7.10.3",
     "@babel/preset-typescript": "^7.10.1",
+    "@playwright/jest-wrapper": "0.0.8",
```

Comment:
What is this?

Reply:
This is from the `playwright-runner` repo. It converts simple error messages/failing tests/etc into the complicated format jest wants to function. It also handles things like filtering tests and wrangling jest's many config options.

Reply:
https://github.com/microsoft/playwright-runner/tree/master/packages/jest-wrapper